scripts/integration-test.sh

```diff
@@ -131,7 +131,7 @@ function cleanup() {
   echo "INFO: Terminating background jobs"
   echo "  any kill usage errors below may be ignored"
 
-  for p in collector processor middleware
+  for p in collector processor
```

Comment:
We still use the middleware in production. I think we should keep this around until we can entirely remove it.